regenerate.py

```diff
@@ -27,10 +28,8 @@
 
 def run(*args) -> str:
   logging.debug('Running command: %s', args)
-  result = subprocess.run(args,
-                          stdout=subprocess.PIPE,
-                          stderr=subprocess.PIPE,
-                          encoding='utf-8')
+  result = subprocess.run(
```

Comment:
@wbthomason I'm a little confused about why the automatic formatting changed. Which way does it format on your machine?

Reply:
The old way, i.e. stdout, etc. are all on their own line. I checked, and I'm running yapf with the repo .style.yapf

Reply:
What version of yapf?

Reply:
`yapf 0.27.0`
Tabstop and shiftwidth are 2.
No change if I try to format the `source` version of this file.
I think 5a5d53d was when I fixed the problem where `yapf` was preferring my personal config, which is:

```
[style]
based_on_style = chromium
indent_width = 2
column_limit = 100
```

Reply:
Okay, this should be fixed now. The 0.27.0 style is not backwards compatible with the 0.22.0 one. I upgraded my version of yapf to 0.27.0.